content/v2/csidriver/features/powerscale.md

```diff
@@ -1,570 +1,659 @@
+weight: 1
+---
+
+## Multicluster support
+
+You can connect a single CSI-PowerScale driver with multiple PowerScale clusters.
+
+**Pre-Requisites:**
+
+1. Creation of secret.yaml with credentials related to one or more Clusters.
+2. Creation of (at least) one Storage class for each cluster.
+3. Creation of custom-volumesnapshot classes with proper isiPath matching corresponding storage classes.
+4. Inclusion of cluster name in volume handle, if you want to provision existing static volumes.
+
+## Consuming existing volumes with static provisioning
+
+You can use existent volumes from the PowerScale array as Persistent Volumes in your Kubernetes, perform the following steps:
+
+1. Open your volume in One FS, and take a note of volume-id.
+2. Create PersistentVolume and use this volume-id as a volumeHandle in the manifest. Modify other parameters according to your needs.
+3. In the following example, the PowerScale cluster accessZone is assumed as 'System', storage class as 'isilon', cluster name as 'pscale-cluster' and volume's internal name as 'isilonvol'. The volume-handle should be in the format of <volume_name>=_=_=<export_id>=_=_=<zone>=_=_=<cluster_name>
+4. If Quotas are enabled in the driver, it is required to add the Quota ID to the description of the NFS export in this format: 
+`CSI_QUOTA_ID:sC-kAAEAAAAAAAAAAAAAQEpVAAAAAAAA`
+5. Quota ID can be identified by querying the PowerScale system.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: isilonstaticpv
+  namespace: default
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: isilon
+  csi:
+    driver: csi-isilon.dellemc.com
+    volumeAttributes:
+        Path: "/ifs/data/csi/isilonvol"
+        Name: "isilonvol"
+        AzServiceIP: 'XX.XX.XX.XX'
+    volumeHandle: isilonvol=_=_=652=_=_=System=_=_=pscale-cluster
+  claimRef:
+    name: isilonstaticpvc
+    namespace: default
+```
+
+3. Create PersistentVolumeClaim to use this PersistentVolume.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: isilonstaticpvc
+  namespace: default
+spec:
+  accessModes:
+  - ReadWriteMany
+  resources:
+        requests:
+          storage: 5Gi
+  volumeName: isilonstaticpv
+  storageClassName: isilon           
+```
+
+4. Then use this PVC as a volume in a pod.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+    name: static-prov-pod
+spec:
+    containers:
+      - name: test
+        image: docker.io/centos:latest
+        command: [ "/bin/sleep", "3600" ]
+        volumeMounts:
+          - mountPath: "/data0"
+            name: pvol
+    volumes:
+      - name: pvol
+        persistentVolumeClaim:
+            claimName: isilonstaticpvc
+```
+
+5. After the pod becomes `Ready` and `Running`, you can start to use this pod and volume.
+
+## PVC Creation Feature
+
+Following yaml content can be used to create a PVC without referring any PV.

```

Comment:
Insert "The" before "Following" and remove initial uppercase from "Following"

Reply:
Addressed the comment